build.sh

```diff
@@ -73,13 +73,23 @@ fi
 
 # check version
 echo "[INFO] Checking VERSION"
+
 if [ -f $shell_dir/VERSION ]; then
   version_file=VERSION
-  version=$(cat $shell_dir/$version_file)
+elif [ -f $shell_dir/output/VERSION-DIST ]; then   
+  version_file=output/VERSION-DIST
+fi
+
+_version=$(cat $shell_dir/$version_file)
+serial_number=$(echo $_version | cut -d . -f 4)
+if [ "x$serial_number" != "x" ]; then
+  version=$_version
+elif [ -d $shell_dir/.git -o -d $shell_dir/../.git ]; then
```

Comment:
$shell_dir/../.git

Can I know why the .git folder of the parent path, `$shell_dir/../.git`, should be checked?

Reply:
The cubrid-jdbc has .git directory as standalone. 
But it has .git 'file' as a submodule which uses the .git of the parent path. 
This is not a directory anymore at that time. 
If the .git directory of the parent path doesn't exist, we can't use git command.

Reply:
Ok, I see